config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml

```diff
@@ -199,6 +199,41 @@ presubmits:
       testgrid-tab-name: pr-secrets-store-csi-driver-e2e-windows
       description: "Run E2E tests on a Windows cluster for Secrets Store CSI driver."
       testgrid-num-columns-recent: '30'
+  - name: pull-secrets-store-csi-driver-e2e-gcp
+    decorate: true
+    decoration_config:
+      timeout: 25m
+    always_run: true
+    path_alias: sigs.k8s.io/secrets-store-csi-driver
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+      # this is required because we want to run kind in docker
+      preset-dind-enabled: "true"
+      # this is required to make CNI installation to succeed for kind
+      preset-kind-volume-mounts: "true"
+      # sets up the gcp parameters used for testing
+      preset-gcp-secrets-store-creds: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20201005-0b243c0-master
+        command:
+        - runner.sh
+        args:
+        - bash
+        - -c
+        - >-
+          apt-get update && apt-get install bats && apt-get install gettext-base -y &&
+          make e2e-bootstrap &&
+          make e2e-gcp
+        securityContext:
+          privileged: true
+    annotations:
+      testgrid-dashboards: sig-auth-secrets-store-csi-driver, provider-gcp-presubmit
```

Comment:
I don't think this dashboard `provider-gcp-presubmit` exists today? We can just keep `sig-auth-secrets-store-csi-driver`

Reply:
nope! this was just copy-pasting errors, removed!